README.md

```diff
@@ -1 +1,8 @@
-Repo for image-jenkins
+## What is the image for?
+The intended purpose of this image is for it to be used as a Jenkins master which will execute Jenkins pipelines by means of different Jenkins agents. This container and its agents are intended to be run via [Kubernetes](https://plugins.jenkins.io/kubernetes/).
+
+For Jenkins agents that are intended to be used with this master, see our various `image-builder` repositories.
```

Comment:
I feel like an example of using this image as a Jenkins master with a image-builder agent would be helpful.